src/types.rs

```diff
@@ -552,6 +552,7 @@ pub enum AuthorizationStatus {
     Invalid,
     Revoked,
     Expired,
+    Deactivated,
```

Comment:
Nit: I don't think this is worth its own commit.

Reply:
Folded into the commit that adds deactivation.